css/styles.css

```diff
@@ -0,0 +1,169 @@
+a {
+  text-decoration: none;
+}
+
+footer {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+}
+
+button {
+  color: white;
+  background-color: rgb(10, 82, 130);
+  border: none;
+  border-radius: 5px;
+  height: 25px;
+  width: 23%;
+}
```

Comment:
Applying this width and height to all buttons could be a problem down the line. You should apply this to a class instead.